powsybl-geo-data-model/src/main/java/extensions/LineGeoData.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Line;
+import infrastructure.LineGraphic;
+
+/**
+ *
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public class LineGeoData extends AbstractExtension<Line> {
+
+    static final String NAME = "line-geo-data";
+
+    private LineGraphic lineGraphic;
```

Comment:
There is a design issue here, because LineGraphic embed redundant information like country, voltage which are already available in the base model (Line). LineGraphic is a DTO object used in REST interface for JSON generation and should not be in the IIDM extension. One more reason to use extensions of GL modules which only contains positions.